tests/aframe.test.mjs

```diff
@@ -85,3 +85,34 @@ test.describe('Navigation', () => {
     expect(title).not.toBe('A-Frame Test');
   });
 });
+
+test.describe('Prebid', () => {
+  async function injectIFrame(page, url, image) {
```

Comment:
Would these tests continue to be relevant if we're not using iframes?

Reply:
Yes, I'm injecting an iframe here to replicate getting an actual ad, since the prebid wrapper can't fetch real ones for testing.